docs/en/operations/settings/permissions_for_queries.md

```diff
@@ -34,25 +34,21 @@ When using the `GET` method in the [HTTP interface](../../interfaces/http.md), `
 Setting `readonly = 1` prohibit the user from changing all the settings. There is a way to prohibit the user
 from changing only specific settings, for details see [constraints on settings](constraints_on_settings.md).
 
-**Default value**
-
-0
+Default value: 0
 
 ## allow_ddl {#settings_allow_ddl}
 
-Allows/denies [DDL](https://en.wikipedia.org/wiki/Data_definition_language) queries.
+Enables or disables [DDL](https://en.wikipedia.org/wiki/Data_definition_language) queries.
```

Comment:
It's not about enabling, it's about disallowing some user to perform DDL, so it should be different from most other settings (like it was in previous version).